example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -239,6 +241,8 @@ describe('The S3 Ingest Granules workflow', () => {
         distribution_endpoint: process.env.DISTRIBUTION_ENDPOINT
       }
     );
+
+    opendapFilePath = `https://opendap.uat.earthdata.nasa.gov/providers/CUMULUS/collections/MODIS/Terra%20Surface%20Reflectance%20Daily%20L2G%20Global%20250m%20SIN%20Grid%20V006/granules/${granuleId}`;
```

Comment:
Since this is just for tests, this is fine as a UAT URL. The URL is `https://opendap.uat.earthdata.nasa.gov/providers/<CMR PROVIDER>/collections/<Encoded collection entry title>/granules/${granuleId}` We set the entry title in our fake processing task when we generate fake CMR data.

Reply:
Did I miss the corresponding update to the fake processing task?

Reply:
Bleh I mispoke, we set the granule Id in the fake processing task. The entry title lives on the collection in the CMR.

Reply:
I'm still not grasping how we can predict this part of the URL: `MODIS/Terra%20Surface%20Reflectance%20Daily%20L2G%20Global%20250m%20SIN%20Grid%20V006`. We have a collection in `example/data/collections/s3_MOD09GQ_006` and then the hyrax task looks up the collection entry title in CMR based on that name/version?

Reply:
That part of the URL is the entry title of the collection the granule belongs to. We do a query of CMR to determine that using the short name and version id of the collection present in the granule metadata.